extension/camunda-spring-boot-starter-webapp/pom.xml

```diff
@@ -20,6 +20,13 @@
       <groupId>org.camunda.bpm.webapp</groupId>
       <artifactId>camunda-webapp-webjar</artifactId>
     </dependency>
+    
+    <dependency>
+      <groupId>org.camunda.bpm.webapp</groupId>
+      <artifactId>camunda-webapp</artifactId>
+      <type>jar</type>
+      <classifier>classes</classifier>
+    </dependency>    
 
```

Comment:
That seems to have changed - in 7.6.0-alpha3-ee this is NOT the case, that's why I added it to make sure it works in every version